imports/plugins/included/shipping-rates/server/no-meteor/schemas/restrictions.graphql

```diff
@@ -0,0 +1,217 @@
+####
+# Custom data types for FlatRateFulfillmentRestriction data
+####
+type attributeRestrictions {
```

Comment:
All types, enums, inputs, etc. should start with a capital letter by GraphQL convention